src/main/java/io/swagslash/gametrackerserver/igdbconsumer/model/IGDBCover.java

```diff
@@ -1,13 +1,13 @@
 package io.swagslash.gametrackerserver.igdbconsumer.model;
 
-public class Cover {
-    boolean alpha_channel;
-    boolean animated;
-    int game; // The game this cover is associated with
-    int height; // The height of the image in pixels
-    String image_id; // The ID of the image used to construct an IGDB image link
-    String url;
-    int width;
+public class IGDBCover {
+    private boolean alpha_channel;
```

Comment:
Please use camel-case
Also below for `image_id`

Reply:
The IGDB models are mapped to the API, so the variable names will have to stay like this. Don't know if I could change the getter/setter but will look into it.